core/tests/protractor_utils/PreferencesPage.js

```diff
@@ -81,6 +88,10 @@ var PreferencesPage = function() {
   // This function only compares the text displayed on the subscription (which
   // might be abbreviated), rather than the text on the popover that appears
   // when hovering over the tile.
```

Comment:
This comment (Line 88-90) is related to `this.expectDisplayedFirstSubscriptionToBe` so you have to move your new check below expectDisplayedFirstSubscriptionToBe.

Reply:
Ok